.buildkite/scripts/links.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -euo pipefail
+
+for file in "$@"; do
+  echo "- <a href=\"${BUILDKITE_ARTIFACT_UPLOAD_DESTINATION}/${file}\">${file}</a>"
```

Comment:
may as well make this a HTTP link since buildkite seems to be stripping this at the moment